webpack/assets/javascripts/react_app/redux/actions/common/forms.js

```diff
@@ -46,7 +46,7 @@ export const submitForm = ({
         dispatch(addToast({
           type: 'success',
           // eslint-disable-next-line no-undef
-          message: Jed.sprintf('%s was successfully created.', __(item)),
+          message: message || Jed.sprintf('%s was successfully created.', __(item)),
```

Comment:
what if you would like to use item in your custom message?

Reply:
I added a test. I think you know `item` already, because you are passing it into the `submitForm`